### PR TITLE
Make CGAL installation relocable

### PR DESCRIPTION
**Pull request by sloriot**
After calling make install, this allow to move the CGAL installation to another directory than the one where it was originally installed

### CODE REVIEW

File: Installation/cmake/modules/CGALConfig_install.cmake.in [hunk 1]
```diff
@@ -33,9 +33,9 @@ set(CGAL_SHARED_LINKER_FLAGS_RELEASE_INIT "@CMAKE_SHARED_LINKER_FLAGS_RELEASE@"
 set(CGAL_SHARED_LINKER_FLAGS_DEBUG_INIT   "@CMAKE_SHARED_LINKER_FLAGS_DEBUG@" )
 set(CGAL_BUILD_TYPE_INIT                  "@CMAKE_BUILD_TYPE@" )
 
-set(CGAL_INCLUDE_DIRS  "@CMAKE_INSTALL_PREFIX@/@CGAL_INSTALL_INC_DIR@" )
-set(CGAL_MODULES_DIR   "@CMAKE_INSTALL_PREFIX@/@CGAL_INSTALL_CMAKE_DIR@" )
-set(CGAL_LIBRARIES_DIR "@CMAKE_INSTALL_PREFIX@/@CGAL_INSTALL_LIB_DIR@" )
+set(CGAL_INCLUDE_DIRS  "${CGAL_INSTALL_PREFIX}/include" )
+set(CGAL_MODULES_DIR   "${CGAL_INSTALL_PREFIX}/lib/CGAL" )
+set(CGAL_LIBRARIES_DIR "${CGAL_INSTALL_PREFIX}/lib" )
```
Comment (lrineau) [inline Installation/cmake/modules/CGALConfig_install.cmake.in]: This implementation completely ignore the following cache variables, defined in `CMakeLists.txt` at the root of the tarball (or in `Installation/`):
```
set ( CGAL_INSTALL_INC_DIR "${CMAKE_INSTALL_INCLUDEDIR}" CACHE STRING "The folder where CGAL header files will be installed, relative to CMAKE_INSTALL_PREFIX" )
set ( CGAL_INSTALL_LIB_DIR "${CMAKE_INSTALL_LIBDIR}"     CACHE STRING "The folder where CGAL libraries will be installed, relative to CMAKE_INSTALL_PREFIX" )
#[...]
set ( CGAL_INSTALL_BIN_DIR   "${CMAKE_INSTALL_BINDIR}"
  CACHE STRING "The folder where CGAL user-side scripts will be installed, relative to CMAKE_INSTALL_PREFIX"
  )

set ( CGAL_INSTALL_CMAKE_DIR "${CGAL_INSTALL_LIB_DIR}/CGAL"
  CACHE STRING "The folder where CGAL CMake modules will be installed, relative to CMAKE_INSTALL_PREFIX" 
  )

set ( CGAL_INSTALL_DOC_DIR "${CMAKE_INSTALL_FULL_DATAROOTDIR}/doc/${CGAL_VERSION_DIR}"
  CACHE STRING "The folder where CGAL documentation and license files will be installed, relative to CMAKE_INSTALL_PREFIX"
  )

set ( CGAL_INSTALL_MAN_DIR "${CMAKE_INSTALL_FULL_DATAROOTDIR}/man/man1"
  CACHE STRING "The folder where manual pages for CGAL scripts will be installed, relative to CMAKE_INSTALL_PREFIX"
  )

**Comment by sloriot** [comment]
@joachim-reichel Any comment on this patch?

**Comment by joachim-reichel** [comment]
I suppose "make install DESTDIR=..." still works? That's what the Debian package is using.

Did you check that the installation prefix does not appear anywhere else? Just wondering whether this change is enough, or whether it works for compiling and linking but some other use case breaks after moving.

**Comment by sloriot** [comment]
@joachim-reichel There is no change in the installation procedure, only a file generated by (well filled by) `cmake` changes. It was the only place were the installation directory was mentioned after a call to `make install`.

**Pull request merged**